ui/src/lib/resultsList/ResultsList.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+	import SvelteTable from 'svelte-table/src/index';
```

Comment:
```suggestion
	import SvelteTable from 'svelte-table';
```

Reply:
If you import like that JEST cant import it